### PR TITLE
Google login

### PR DESCRIPTION
**Pull request by JessSaavedra**
## Summary

- Added Google Login

## Trello Card

https://trello.com/c/uWeDQpkR

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -34,4 +34,6 @@ bower.json
 # Ignore pow environment settings
 .powenv
 
-.mail.yml
\ No newline at end of file
+.mail.yml
+
+config/initializers/01_development_env.rb
```
Comment (epintos) [inline .gitignore]: What is this?

File: config/initializers/devise.rb [hunk 2]
```diff
@@ -262,4 +262,5 @@
   # When using OmniAuth, Devise cannot automatically set OmniAuth path,
   # so you need to do it manually. For the users scope, it would be:
   # config.omniauth_path_prefix = '/my_engine/users/auth'
+  config.omniauth :google_oauth2, ENV['GOOGLE_CLIENT_ID'], ENV['GOOGLE_CLIENT_SECRET']
```
Comment (epintos) [inline config/initializers/devise.rb]: Why arent you using secrets or app configuration? Are you using sth else?

**Comment by JessSaavedra** [comment]
Fixed in facebookLogin

**Comment by JessSaavedra** [comment]
Merged manually

**Pull request closed without merging**